lms/djangoapps/mobile_api/testutils.py

```diff
@@ -202,6 +205,7 @@ def test_unenrolled_user(self):
         (None, False)
     )
     @ddt.unpack
+    @patch.dict(settings.FEATURES, {'ENABLE_MKTG_SITE': True})
     def test_visible_to_staff_only_course(self, role, should_succeed):
         self.init_course_access()
         self.course.visible_to_staff_only = True
```

Comment:
@jcdyer The tests for utility method covers all possible scenarios. Here https://github.com/edx/edx-platform/pull/12818/files#diff-09a11b1ae6b6f1e1407c364af13874e4R9

Do we repeat these scenarios everywhere again?